e2e/cypress/integration/enterprise/guest_accounts/member_invitation_ui_spec.js

```diff
@@ -17,13 +17,15 @@ let testTeam;
 const user1 = users['user-1'];
 const sysadmin = users.sysadmin;
 
-function invitePeople(typeText, resultsCount, verifyText) {
+function invitePeople(typeText, resultsCount, verifyText, skipInitialStep = false) {
     // # Open Invite People
     cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
     cy.get('#invitePeople').should('be.visible').click();
 
-    // #Click on the next icon to invite new member
-    cy.findByTestId('inviteMembersLink').find('.arrow').click();
+    if (!skipInitialStep) {
```

Comment:
Hi @jespino 
Any reason why you had chosen to skipInitialStep. Currently the test fails because it's unable to click on the next arrow in the initial step. If I remove this step and always click on the next step, the testcase passes. Can you please revert the corresponding code across related to skipInitialStep.

Reply:
Wasn't working for me locally, I don't know exactly why. But yes, I can revert that part of the chagnes.

Reply:
Done, PTAL